client/index.html

```diff
@@ -12,8 +12,51 @@
     <link rel="shortcut icon" href="app/icon/carrot_logo.ico">
     <link rel="icon" type="image/png" href="app/icon/carrot_logo.png" sizes="32x32">
     <base href="/">
+    <style type="text/css">
```

Comment:
i put in intentionally in the index.html, that the loading spinner is avaiable as soon as the first data is loaded ;)